### DISCUSSION TITLE
NEW MEMBER

### DISCUSSION THREAD

**Discussion by Kd502020** [Cyfrin/foundry-full-course-cu#1730] [General]
hello world🙌

**Comment by Kd502020** [comment]
JUST COMPLETED MY BIGINNERS LESSON AND I'M HAPPY TO BE HERE

**Reply by EngrPips** [reply]
Welcome Aboard @Kd502020.

**Reply by Alexandre-Jeandel** [reply]
i am starting the first lesson today. Good job to you and good luck

**Reply by EngrPips** [reply]
I wish you the best of luck on your journey, Friend.
